不生成 /tmp/1.txt

qnnnnez opened the pull request:
`ok-connect` 在运行的时候会将一个变量的值输出到 `/tmp/1.txt`。
但是这个文件在别的地方没有被使用到，大概是作者调试脚本之后忘了删除。

qnnnnez committed:
do not create /tmp/1.txt

Eventide commented:
感谢提出这个PR！
实际上在这行下面的
```sh
echo $OVERRIDE_PPPD_COMMAND
```
也是可以移除/注释的。
你可以再追加修改。

qnnnnez committed:
remove a useless output line

qnnnnez commented:
已经删除那一行。

qnnnnez committed:
修复有时候无法连接的问题

有时候加密后的用户名中会含有引号，这种情况下会导致拨号失败。
现在我用命令行参数来传递用户名和密码，避开了临时文件。
为了直接把参数传递给 pppd，我把命令行参数展开，传递给
pppd，而不通过变量暂存。
